include/multipass/logging/log.h

```diff
@@ -29,58 +29,43 @@ namespace logging
 {
 
 /**
- * Log a message.
+ * Log a preformatted message.
  *
  * It's safe to use this function with non-NUL terminated strings.
  *
  * @param [in] level Log level
  * @param [in] category Log category
  * @param [in] message The message
  */
-void log(Level level, std::string_view category, std::string_view message);
+void log_message(Level level, std::string_view category, std::string_view message);
```

Comment:
We discussed this in the standup, but for anyone who wasn't there, here's the backstory:

In C++20, the `fmt` library can do compile-time checking of format strings. One feature of the compile-time checking is that it reports an error if you have more format placeholders than args. That's good for us, since it caught a few bugs, but it means that we'd need some way to explicitly say, "Log this message without any formatting." Now, we can just use `log_message` for that.

Reply:
@xmkg My perspective on `fmt::runtime` is that we should treat it as a marker that means, "Be careful here!" Sort of like a Rust `unsafe` block; like with `unsafe`, `fmt::runtime` is a marker that tells us we're losing compile-time checks that verify our code. As of my most recent push, outside of tests, we're only using `fmt::runtime` in 3 places in the code, and those are all passing a format string over RPC (I think) in `daemon.cpp`.

Reply:
After some thought, I believe you're right. The fact that we're doing the formatting part unconditionally (not gated behind log level) should reveal any issues assuming we have unit test coverage for all log calls. I've also reviewed the sections where we use fmt::runtime, and they look reasonable to me.